spread.yaml

```diff
@@ -601,26 +609,30 @@ restore-each: |
     if journalctl -u snapd.service | grep -F "signal: terminated"; then exit 1; fi
 
 suites:
-    tests/main/:
-        summary: Full-system tests for snapd
+    # The essential tests run designed to run inside the autopkgtest
```

Comment:
the first "run" seems superfluous:  tests designed to run

Reply:
done